ml-agents/mlagents/trainers/components/reward_signals/gail/model.py

```diff
@@ -2,6 +2,7 @@
 
 from mlagents.tf_utils import tf
 
+from mlagents.trainers.tf_policy import TFPolicy
 from mlagents.trainers.models import LearningModel
```

Comment:
LearningModel is now more of a Utility class. Maybe it should be renamed as such.